test/conformance/platform/urInit.cpp

```diff
@@ -10,7 +10,12 @@ INSTANTIATE_TEST_SUITE_P(
                       UR_DEVICE_INIT_FLAG_VPU,
                       /* Combinations */
                       UR_DEVICE_INIT_FLAG_GPU | UR_DEVICE_INIT_FLAG_CPU,
-                      UR_DEVICE_INIT_FLAG_FPGA | UR_DEVICE_INIT_FLAG_VPU));
+                      UR_DEVICE_INIT_FLAG_FPGA | UR_DEVICE_INIT_FLAG_VPU),
+    [](const ::testing::TestParamInfo<ur_device_init_flags_t> &info) {
+        std::stringstream ss;
+        ur_params::serializeFlag_ur_device_init_flags_t(ss, info.param);
```

Comment:
I didn't expect the serializeFlag functions to be used outside of the params file ;) It's fine, but I think we can make it nicer with a template like this:
`serializeFlag<ur_device_init_flag_t>(ss, info.param);`

But leave it as-is for now. I'll refactor this alongside other changes.

Reply:
I agree - templates is probably the better solution here 👍